operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1230,57 +1230,57 @@ memcached-metadata:
     annotations: {}
     labels: {}
 
-memcached-results:
```

Comment:
I think this configuration is much clearer from a product perspective since it maps directly back to the documentation and configuration parameters.

I'm wondering if I should add a validation to check for the old `memcached.enabled`, `memcached-metadata.enabled`, and `memcached-queries.enabled` and error out if the user hasn't migrated to these names?